### ISSUE TITLE
I found the CxScheme for "ATRI -My Dear Moments-"

### ISSUE THREAD

**Issue by crskycode** [morkt/GARbro#425]
CxScheme [Code Here](https://gist.github.com/crskycode/40070aa666b81da57907df433380bb56)
Update the database please :)

**Comment by imKota** [comment]
Hello @crskycode do you have updated database with support ATRI?

**Comment by imKota** [comment]
@morkt please update the database

**Comment by robonxt** [comment]
@crskycode Could you explain how to get this working? (Sorry, first time using this tool)

**Comment by esperz2019** [comment]
Hello, I manually merged the CxScheme information for *ATRI -My Dear Moments-* from @crskycode into the database, and it is tested working in my case. Would anyone like to give it a shot?

Updated `Formats.dat` is here: https://github.com/esperz2019/GARbro/blob/ad53926dd1bc59a8adaa941116ffcd2982cd513d/ArcFormats/Resources/Formats.dat

**Comment by crskycode** [comment]
> Hello, I manually merged the CxScheme information for *ATRI -My Dear Moments-* from @crskycode into the database, and it is tested working in my case. Would anyone like to give it a shot?
> 
> 
> 
> Updated `Formats.dat` is here: https://github.com/esperz2019/GARbro/blob/ad53926dd1bc59a8adaa941116ffcd2982cd513d/ArcFormats/Resources/Formats.dat

nice work

**Comment by robonxt** [comment]
> Hello, I manually merged the CxScheme information for _ATRI -My Dear Moments-_ from @crskycode into the database, and it is tested working in my case. Would anyone like to give it a shot?

Testing it out right now. Will report back

**Comment by esperz2019** [comment]
> Testing it out right now. Will report back.
> 
> EDIT: Images are working. Audio (voice, sound effects, music) are valid. Videos come out as amv, not sure how to convert it yet.
> tl;dr: It works, thank you!

glad it helps ;)

**Comment by crskycode** [comment]
> CxScheme [Code Here](https://gist.github.com/crskycode/40070aa666b81da57907df433380bb56)
> Update the database please :)

Use this mod [Link](https://github.com/crskycode/GARbro/releases)
